### PR TITLE
Add disabled prop to Stepper

### PR DESCRIPTION
**Pull request by mcellier**
## Description

For trip edit, we need to be able to disable the stepper (i.e., the driver can't update the seat count if a booking is in progress).
We could cheat by setting min/max values equal to the selected value, but MEH.

## What has been done

Added `disabled` prop to the `Stepper` component

## Things to consider

Reusing the disabled display we have when hitting min/max values

## How it was tested

Storybook on chrome

### CODE REVIEW

File: src/stepper/Stepper.tsx [hunk 1]
```diff
@@ -230,11 +235,14 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
...
             className="kirk-stepper-increment"
             status={ButtonStatus.UNSTYLED}
-            disabled={isMax}
+            disabled={disabled || isMax}
             isBubble
             {...this.createButtonListeners(this.increment)}
           >
-            <PlusIcon iconColor={isMax ? color.lightGray : color.blue} size={buttonSize} />
+            <PlusIcon
+              iconColor={disabled || isMax ? color.lightGray : color.blue}
```
Comment (Tuxosaurus) [inline src/stepper/Stepper.tsx:243]: Icons all have the "isDisabled" (I know) prop that should be enough to gray them out

Comment (mcellier) [reply src/stepper/Stepper.tsx:243]: Can't find the code related to that. Can you point it out?

Comment (Tuxosaurus) [reply src/stepper/Stepper.tsx:243]: https://github.com/blablacar/ui-library/blob/master/src/_utils/icon/index.tsx#L14

Comment (mcellier) [reply src/stepper/Stepper.tsx:243]: Thx ! I'll try to clean this a bit

Comment (mcellier) [reply src/stepper/Stepper.tsx:243]: The difference: disabled icon is gray, disabled stepper icon is lightGray 

```
  gray: '#DDD',
  lightGray: '#EDEDED',
```

Is it an issue or can we just put gray everywhere?

Ping @wakooka

Comment (mcellier) [reply src/stepper/Stepper.tsx]: Now using `isDisabled` from icons

**Pull request merged**